### ISSUE TITLE
[TimeSeries] Ensure data is saved and events are published only once

### ISSUE THREAD

**Issue by lasrinnil** [Energinet-DataHub/geh-timeseries#142]
**Description**
There must be cohesion between saving data and publishing events.

**Tech notes**
Look up the "Outbox pattern" for inspiration and talk with Kristian Fauerskov

**Acceptance criteria**
1) Data must only be saved once
2) Events must only be published if data is saved
3) Events must be published and only be published once

**Comment by prtandrup** [comment]
Closed. Duplicate of #141. A similar issue has been created in the charges repo, look for # 400.
